### ISSUE TITLE
[Feature Request] Allow reserved users to remove themselves

### ISSUE THREAD

**Issue by Shwaffle** [sillvva/rpg-schedule#106]
As it currently stands, if you reserve a person onto an event they are unable to react to the **-** to remove themselves. We use this for clan events and I would like to auto-reserve raid leaders but allow them to remove themselves if needed.

Since it's tied to the users Discord account(User#1234), is there a way that it can realize that user is submitting a removal action?

**Comment by sillvva** [comment]
If you're entering their discord names for them, it should allow them to use the reaction to remove themselves if their tag was spelled correctly.

**Comment by Shwaffle** [comment]
So I went back and tested it this morning and I'm getting mixed results. I put a total of 6 users on the list and only two of us are able to remove ourselves.

I am using Discord ID's, and all roles have permissions to post/edit events.

This is how our roles were ordered beforehand:

Admin (Admin flag) [can remove]
Moderator (No admin flag) [can remove]
Bot (Admin flag)
Raid Leader (No admin flag) [cannot remove]

After your response I moved Raid Leader above the bot and tested again. They still cannot remove themselves. Is there any settings I should be double checking?


Thanks for your prompt responses. Love the bot! I've got another feature request I'll post this weekend for consideration.
